editor/src/clj/editor/workspace.clj

```diff
@@ -467,17 +467,20 @@ ordinary paths."
   (or (:template resource-type)
       (some->> resource-type :ext (str "templates/template."))))
 
+(def ^:private user-resource-path "/templates/template.")
+
 (defn- get-template-resource [workspace resource-type]
   (let [path (template-path resource-type)
         java-resource (when path (io/resource path))
+        user-resource (find-resource workspace (some->> resource-type :ext (str user-resource-path)))
         editor-resource (when path (find-resource workspace path))]
-    (or java-resource editor-resource)))
-  
+    (or user-resource java-resource editor-resource)))
```

Comment:
This is actual change.  The order priority for different template PATHs:
if user path in the project exist - use this one, if not use one from java, if not check if we have a special path specified (for extensions)

Reply:
I think it would be nice if we could avoid unnecessary lookups. For example, we do `editor-resource` lookup even if `user-resource` exists. I'm suggesting something like:
```clj
(or 
  ;; user resource
  (find-resource workspace (some->> resource-type :ext (str user-resource-path)))
  ;; java resource
  (when path (io/resource path))
  ;; editor resource
  (when path (find-resource workspace path)))
```

Reply:
true

Reply:
I go further and modified it a little bit more (tested with spine extension to make sure it forks)